code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -62,7 +66,8 @@
 			else if(E.status & ORGAN_BROKEN)
 				tally += 1.5
 
-	if(shock_stage >= 10) tally += 3
+	if(shock_stage >= 10 || get_stamina() <= 0)
```

Comment:
can stamina actually go below zero?

change this value to something useful, like under 30

Reply:
also can you use the direct number rather than calling a proc here? this proc is called with every single step a human takes, optimising is important

Reply:
> can stamina actually go below zero?

There is nothing preventing stamina going below 0 except the fact that you cannot run while you have 0 stamina.

> change this value to something useful, like under 30

This is used to make people move slower when they've spent all their stamina.